packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -271,16 +273,21 @@ const nextServerlessLoader: loader.Loader = function() {
           `
             : `const nowParams = null;`
         }
+        renderOpts.params = _params || params
+
         let result = await renderToHTML(req, res, "${page}", Object.assign({}, unstable_getStaticProps ? {} : parsedUrl.query, nowParams ? nowParams : params, _params), renderOpts)
 
         if (_nextData && !fromExport) {
           const payload = JSON.stringify(renderOpts.pageData)
           res.setHeader('Content-Type', 'application/json')
           res.setHeader('Content-Length', Buffer.byteLength(payload))
-          res.setHeader(
-            'Cache-Control',
-            \`s-maxage=\${renderOpts.revalidate}, stale-while-revalidate\`
-          )
+
+          if (renderOpts.revalidate) {
```

Comment:
Should we set explicit no cache headers `no-cache, no-store, must-revalidate`?

Reply:
Is there any reason we didn't add the `no-cache, no-store, must-revalidate` here?

Reply:
Added